app/graphql/types/system.rb

```diff
@@ -30,7 +30,7 @@ class System < Types::BaseObject
 
     def profiles(policy_id: nil)
       context_parent
-      all_profiles = object.all_profiles
+      all_profiles = object.policy_profiles
```

Comment:
We cannot restrict it to only one set of profiles without breaking the API and UI. The API here has to report all system's profile, including `external: true` policy profiles that the system is reporting under. Each of the returned profile caries its own result of a report (passed/failed rules). That's by design.

Reply:
Here is one of the reason: https://github.com/RedHatInsights/compliance-frontend/pull/832#discussion_r526327011